### DISCUSSION TITLE
Trying to nest non-velocity fields from two ROMS and add it to the FieldSet

### DISCUSSION THREAD

**Discussion by jwongala** [Parcels-code/Parcels#1545] [Q&A]
Hello! I want to use NestedFields to nest the non-velocity fields (e.g., temperature) from two different ROMS ([main Hawaiian Islands](https://pae-paha.pacioos.hawaii.edu/erddap/griddap/roms_hiig.html) and [Oahu south shore](https://pae-paha.pacioos.hawaii.edu/erddap/griddap/roms_hiomsg.html)) and add it to the FieldSet. I have been working on this but have yet to make much headway. If anyone could provide any help or advice, I would greatly appreciate it! 

Mahalo,
Jennifer

**Comment by erikvansebille** [comment]
Hi @jwongala, thanks for your request for advice; but we'd need a bit more information. What is not working? What issue are you running into?

**Reply by jwongala** [reply]
hi apologies the issue is fixed. I had to update parcels and now I can nest non-velocity fields.
